ydb/core/blobstorage/vdisk/huge/blobstorage_hullhugeheap.cpp

```diff
@@ -408,32 +408,27 @@ namespace NKikimr {
                 const TString &vdiskLogPrefix,
                 ui32 chunkSize,
                 ui32 appendBlockSize,
-                ui32 minHugeBlobInBytes,
+                ui32 minBlobInBytes,
                 ui32 milestoneBlobInBytes,
                 ui32 maxBlobInBytes,
                 ui32 overhead,
                 bool oldMapCompatible)
             : VDiskLogPrefix(vdiskLogPrefix)
             , ChunkSize(chunkSize)
             , AppendBlockSize(appendBlockSize)
-            , MinHugeBlobInBytes(minHugeBlobInBytes)
+            , MinBlobInBytes(minBlobInBytes)
```

Comment:
What's the point of removing "Huge" infix from the identifier?

Reply:
I think it's more consistent with the left boundary naming. Now the range is presented by [MinBlobInBytes, MaxBlobInBytes].

Reply:
But the "min blob size" is 1 byte. And the "min huge blob size" is absolutely different metric.
It looks like
1b <-- small blobs --> minHugeBlobSize <-- huge blobs --> maxBlobSize